src/platforms/posix/px4_layer/px4_posix_tasks.cpp

```diff
@@ -65,6 +65,15 @@
 #define PX4_MAX_TASKS 50
 #define SHELL_TASK_ID (PX4_MAX_TASKS+1)
 
+#if defined(__PX4_POSIX_RPI)
+/**
+ * The minimum stack size of 16384 bytes on Debian Jessie for the Raspberry Pi is too small.
+ * We add the minimum amount possible for it to function properly, in this case 4K.
+ */
+#undef PTHREAD_STACK_MIN
+#define PTHREAD_STACK_MIN 16384 + 4096
```

Comment:
Yeah I'm not a fan of undef's myself, but seemed to be appropriate for this specific case. Other option would be replace usage of `PTHREAD_STACK_MIN` with `#define PX4_POSIX_STACK_MIN PTHREAD_STACK_MIN + 4096`. 

Let me know if you need me to test out your changes on my system.